### ISSUE TITLE
Support multiarch docker images

### ISSUE THREAD

**Issue by sontek** [zapier/prom-aggregation-gateway#46]
Currently we only support amd64:

```
❯ docker image inspect ghcr.io/zapier/prom-aggregation-gateway:v0.5.0|grep -i arch
        "Architecture": "amd64",
```

We should publish arm images as well.

**Comment by cwboden** [comment]
Not sure if discussion is preferred here or in the PR, but I took a stab at adding multi-arch support in #81. 

Hoping for a merge soon, as we've recently migrated to `arm64` nodes, but still would like to use the push gateway! 😁

**Comment by cwboden** [comment]
This issue can now be closed, since #81 has been merged :)

**Comment by cwboden** [comment]
@mplachter Does something need to be done to run the deployment pipeline? I would like to see a new artifact with the fix, but one hasn't kicked off since #81 was merged

**Comment by CallumD** [comment]
I also need this (thanks @cwboden) so would love to see the build run. Possibly @djeebus can assist?

**Comment by djeebus** [comment]
Apologies! Thanks for the ping, let me get this released.

**Comment by yafanasiev** [comment]
@djeebus we are psyched to see this release, but it looks like release pipeline had failed

**Comment by djeebus** [comment]
Yup, I've got the [release pipeline fixed](https://github.com/zapier/prom-aggregation-gateway/pull/83) (probably), but there's something odd going on with the skaffold build. Still working on it. Stay tuned!

**Comment by djeebus** [comment]
Well that was fun and strange. We're good now though! v0.9.0 should be a multiarch image, tested on x86 and an apple m2. Have fun!
